styles.css

```diff
@@ -12,7 +8,11 @@ body {
     grid-gap: 16px;
     grid-template-columns: 1fr 1fr 1fr;
     grid-template-rows: 50px 100px auto min-content 50px;
```

Comment:
de min-content werkt alleen moet je hierdoor je height defineren bij .d (dit is lastig als je dezelfde wil gebruiken voor je mobiele versie) Je had hem zo kunnen doen 50px 190px 100px auto 50px en dan bij de grid-template-areas, de vierde regel een . ipv een d.